PCbuild/readme.txt

```diff
@@ -293,3 +293,23 @@ project, with some projects overriding certain specific values. The GUI
 doesn't always reflect the correct settings and may confuse the user
 with false information, especially for settings that automatically adapt
 for different configurations.
+
+Add a new project
+-----------------
+
+For example, add a new _testclinic_limited project to build a new
+_testclinic_limited extension, the file Modules/_testclinic_limited.c:
+
+* In PCbuild/, copy _asyncio.vcxproj to _testclinic_limited.vcxproj,
```

Comment:
I'd suggest copying an existing `test` project if you're adding a test project, or copy an existing library module if you're adding a library module.

The main difference is whether they get PGO or not, but also whether a PGO failure will break the entire build.